group-actors/src/main/java/org/sunbird/actors/BaseActor.java

```diff
@@ -63,10 +63,10 @@ public void onReceive(Object message) throws Throwable {
    */
   protected void onReceiveException(String callerName, Exception exception) throws Exception {
     logger.error(
-            "Exception in message processing for: {} :: message: {} {}"
-            , callerName
-            , exception.getMessage(),
-            exception);
+        "Exception in message processing for: {} :: message: {} {}",
```

Comment:
please add context in logger all files

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId